cmd/controller-manager/main.go

```diff
@@ -70,8 +69,7 @@ func main() {
 		}
 	}
 
-	// The default klog flush interval is 30 seconds, which is frighteningly long.
-	go wait.Until(klog.Flush, *logFlushFreq, wait.NeverStop)
+	klog.StartFlushDaemon(*logFlushFreq)
 	defer klog.Flush()
```

Comment:
Is this still needed?

Reply:
Actually, I think it can be removed with `--log-flush-frequency` flag together. 
This is just to preserve the `--log-flush-frequency` flag for compatibility.
Given that all components provide this flag, I think we can just keep it, it's harmless.

Reply:
Got that~